tools/xcode/templates/SFML/SFML App.xctemplate/TemplateInfo.plist.in

```diff
@@ -146,7 +146,7 @@ subject to the following restrictions:
 # If you're using static libraries (which is not recommended) you should remove this script from your project.
 
 # SETTINGS
-CMAKE_INSTALL_FRAMEWORK_PREFIX="@CMAKE_INSTALL_FRAMEWORK_PREFIX@"
+CMAKE_INSTALL_FRAMEWORK_PREFIX="@CMAKE_INSTALL_PREFIX@"
```

Comment:
This file is slightly incorrect. Could you update it with [this one](https://gist.githubusercontent.com/mantognini/9e4e26ce154869c1a246826f96531517/raw/e78e3eafe3eb227fe7f4d14659a83754607571f2/gistfile1.txt)? Thanks :)

Reply:
I suppose you mean rename the variable CMAKE_INSTALL_FRAMEWORK_PREFIX to something else?